account-creation-service/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"golang.org/x/crypto/bcrypt"
+	"github.com/joho/godotenv"
+)
+
+type User struct {
+	Email     string `json:"email" bson:"email"`
+	Name  string `json:"name" bson:"name"`
+	Password  string `json:"password" bson:"password"`
+	CreatedAt time.Time `bson:"createdDate"`
+	Type string `bson:"accountType"`
+}
+
+var userCollection *mongo.Collection
+
+func connectDB() *mongo.Client {
```

Comment:
Should this be a separate folder under DB client?

Reply:
Donezo